scripts/scil_run_commit.py

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Convex Optimization Modeling for Microstructure Informed Tractography (COMMIT)
+estimates, globally, how a given tractogram explains the DWI in terms of signal
+fit, assuming a certain forward microstructure model. It assigns a weight to
+each streamline, which represents how well it explains the DWI signal globally.
+The default forward microstructure model is stick-zeppelin-ball, which requires
+multi-shell data and a peak file (principal fiber directions in each voxel,
+typically from a field of fODFs).
+
+It is possible to use the ball-and-stick model for single-shell data. In this
+case, the peak file is not mandatory.
+
+The output from COMMIT is:
+- fit_NRMSE.nii.gz
+    fiting error (Normalized Root Mean Square Error)
+- fit_RMSE.nii.gz
+    fiting error (Root Mean Square Error)
+- results.pickle
+    Dictionary containing the experiment parameters and final weights
+- compartment_EC.nii.gz (Extra-Cellular)
+- compartment_IC.nii.gz (Intra-Cellular)
+- compartment_ISO.nii.gz (isotropic volume fraction (freewater comportment))
+    Each of COMMIT compartments
```

Comment:
Can you add the essential.trk and non_essential.trk in the list here and
commit_weights.txt